tests/test_pytorch_models.py

```diff
@@ -155,27 +155,69 @@ def test_preview(tmpdir, net):
     assert type(tiledb_obj.preview()) == str
 
 
-def test_function_calls_in_tiledb_cloud_case(tmpdir, mocker):
-    mock_1 = mocker.patch("tiledb.ml._cloud_utils.get_s3_prefix", return_value="")
-    mock_2 = mocker.patch(
-        "tiledb.ml.models.base.TileDBModel.set_file_properties", return_value={}
-    )
-
+@pytest.mark.parametrize(
+    "net",
+    [
+        getattr(sys.modules[__name__], name)
+        for name, obj in inspect.getmembers(sys.modules[__name__])
+        if inspect.isclass(obj) and obj.__module__ == __name__
+    ],
```

Comment:
Mentioned this in another PR, to me these 3 lines is an obscure brittle equivalent of the shorter and clearer `[Net, SeqNeuralNetwork, ConvNet]` (bonus: it's copy-pasted 5 times)

Reply:
The 5-times repetition is due to the fact that the tests are not under a `Testclass`. If you use a `TestClass` wrapping all the tests it should be fine with only one definition of `parameterize` on top of the class. About the model class list I get that as it is now might be a little bit obscure, it depends though on the future plans of how we will test models. If we stick only to the defined models `[Net, SeqNeuralNetwork, ConvNet]`, which are hard-coded and defined by us then George's recommendation for a shorter and clearer definition is more than fine to me.